server/server.go

```diff
@@ -136,6 +136,10 @@ func parseVersionByte(w http.ResponseWriter, r *http.Request) (byte, error) {
 	switch versionByte[0] {
```

Comment:
any reason you need to switch on the version byte, if you're just returning it in each case?

Reply:
nope